content/administration/maintain/google_oauth.rst

```diff
@@ -38,10 +38,22 @@ you are using a personal Gmail account, then leave the :guilabel:`Location` as `
    :align: center
    :alt: Project Name and Location for Google Oauth.
 
-Click on :guilabel:`Create` to finish this step. On the next screen, :guilabel:`User Type` options,
-select the :guilabel:`External` option, and then click on :guilabel:`Create` again, which will
+Click on :guilabel:`Create` to finish this step.
+
+OAuth consent screen
+--------------------
+
+Click on :guilabel:`OAuth consent screen` in the left menu, under :guilabel:`User Type` options,
+select the appropriate :guilabel:`User Type`, and then click on :guilabel:`Create` again, which will
 finally navigate to the :guilabel:`Edit app registration` page.
 
+.. warning::
+   Personal Gmail accounts are only allowed to be :guilabel:`External` User Type, which means Google
+   may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
+   account allows for :guilabel:`Internal` User Type. It should also be noted, that while the API
+   connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no approval is necessary
+   from Google. User limits in this *testing* mode is set to 100 users.
```

Comment:
@tiku-odoo can you take another look here around **guilabel** usage?

It *seems* that some of these should be in **bold** or _italics_ since they appear to:
- be referred to theoretically, and not through active UI instruction
- not be the actual names of headings, buttons, fields, values, etc. on the UI

As well, check capitalization since none of these appear to be proper nouns or product names, i.e. is **External** User Type supposed to be capitalized? Is this unique to Google?

I'm not saying there's a clear right answer for every case, however with the way this is written it's difficult to decipher. You might benefit from changing the language around slightly to _force_ specific RST use more clearly.